arithmetic_analysis/gaussian_elimination.py

```diff
@@ -33,11 +33,11 @@ def retroactive_resolution(
 
     x: NDArray[float64] = np.zeros((rows, 1), dtype=float)
     for row in reversed(range(rows)):
-        sum = 0
+        count = 0
```

Comment:
Count is 1, 2, 3.  Let's use `total` instead.

Reply:
Sure